### PR TITLE
Changes from hackathon #2

### PR DESCRIPTION
**Pull request by alxdavids**
- reworking index page
- working towards standardised bib
- work on properties

Co-authored-by: Michele Orrù <mmaker@users.noreply.github.com>
Co-authored-by: George Kadianakis <asn-d6@users.noreply.github.com>

cc @mmaker, @asn-d6, still in progress

### CODE REVIEW

**Comment by asn-d6** [comment]
I can merge this whenever you want.

**Review by chris-wood — APPROVED** [review]

File: properties.md [hunk 1]
```diff
@@ -52,44 +55,60 @@ Unlinkability, also known as *blindness*, or *context-hiding* in other works,
...
+that permit multi-show operations (such as [[CL06]]({{site.baseurl}}/bibliography.html#bcckls08))
 
 ### Revocation
 
 Defined here by [Acar et al.](https://www.iacr.org/archive/pkc2011/65710436/65710436.pdf) and
 implemented using an accumulator scheme with delegatable non-membership proofs.
 Revocation via blocklisting was also implemented by [Tsang et al.](https://www.cs.dartmouth.edu/~sws/pubs/akts07.pdf).
 
-### Public Attributes
+### Associated Data
```
Comment (chris-wood) [inline properties.md:103]: How is this different from public attributes?

Comment (alxdavids) [reply properties.md:103]: We've made `Public Attributes` a subsection of `Associated data`, mainly to bring in the unlinkability considerations together with this and the private attributes subsection.

**Comment by alxdavids** [comment]
@asn-d6 see what @mmaker thinks. I'm happy for it to be merged so that everyone is working from the same version

**Comment by asn-d6** [comment]
Merged it so that I can work on top of it this week! Thanks all!

**Pull request merged**